src/Persistence.ApplicationState/ApplicationStateViewModelReader.cs

```diff
@@ -1,45 +1,48 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Web;
-using CR.ViewModels.Core;
+﻿// <copyright file="ApplicationStateViewModelReader.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
 
 namespace CR.ViewModels.Persistence.ApplicationState
 {
+    using System;
+    using System.Linq;
+    using System.Web;
+    using Core;
+
+    /// <inheritdoc />
+    /// <summary>
+    /// A implementation of the <see cref="T:CR.ViewModels.Core.IViewModelReader" /> interface that uses
+    /// HttpApplicationState from System.Web to retrieve view models.
+    /// </summary>
     public class ApplicationStateViewModelReader : IViewModelReader
     {
-        private HttpApplicationStateBase AppState { get; set; }
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationStateViewModelReader"/> class, using a provided <see cref="HttpApplicationStateBase"/> to access view models.
```

Comment:
Another lost comment.
We should consistently use the `T:CR.ViewModels.Core.IViewModelWriter` format if we're going to use it in the docs.